e2e/regression/regression_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//go:build e2e
+
+package regression
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/edgelesssys/contrast/e2e/internal/contrasttest"
+	"github.com/edgelesssys/contrast/e2e/internal/kubeclient"
+	"github.com/edgelesssys/contrast/internal/kuberesource"
+	"github.com/edgelesssys/contrast/internal/manifest"
+	"github.com/edgelesssys/contrast/internal/platforms"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	imageReplacementsFile, namespaceFile, platformStr string
+	_skipUndeploy                                     bool // just here for interoptability, ignored in this test
+)
+
+func TestRegression(t *testing.T) {
+	yamlDir := "./e2e/regression/test-data/"
+	files, err := os.ReadDir(yamlDir)
+	require.NoError(t, err)
+
+	platform, err := platforms.FromString(platformStr)
+	require.NoError(t, err)
+
+	runtimeHandler, err := manifest.RuntimeHandler(platform)
+	require.NoError(t, err)
+
+	for _, file := range files {
+		t.Run(file.Name(), func(t *testing.T) {
+			require := require.New(t)
+
+			c := kubeclient.NewForTest(t)
+			ct := contrasttest.New(t, imageReplacementsFile, namespaceFile, platform, false)
+			ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+			defer cancel()
+
+			resources := kuberesource.CoordinatorBundle()
+
+			yaml, err := os.ReadFile(yamlDir + file.Name())
+			require.NoError(err)
+			yaml = bytes.ReplaceAll(yaml, []byte("REPLACE_NAMESPACE"), []byte(ct.Namespace))
```

Comment:
```suggestion
			yaml = bytes.ReplaceAll(yaml, []byte("@@REPLACE_NAMESPACE@@"), []byte(ct.Namespace))
```
Placeholders often use `@@` to be easier to spot.